nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -76,7 +76,7 @@
  * objects stored in them to assist with namespace-centric catalog exploration.
  * </p>
  */
-public class NessieCatalog extends BaseMetastoreCatalog implements AutoCloseable, SupportsNamespaces, Configurable {
+public class NessieCatalog extends BaseMetastoreCatalog implements SupportsNamespaces, Configurable {
```

Comment:
@nastra and @rymurr, is this change okay for Nessie?

Reply:
Thanks @rdblue for helping ping, i think it's equivalent?

Reply:
yep that seems ok, thanks for the ping